src/HonzaBotner.Services.Contract/HonzaBotner.Services.Contract.csproj

```diff
@@ -1 +1,8 @@
-<Project Sdk="Microsoft.NET.Sdk" />
+<Project Sdk="Microsoft.NET.Sdk">
+  <ItemGroup>
+    <ProjectReference Include="..\HonzaBotner.Database\HonzaBotner.Database.csproj" />
```

Comment:
I don't like this direct dependency to Database project. Contract projects should contain only contracts (APIs) and not services (which database is)